Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean

```diff
@@ -1105,6 +1105,12 @@ instance Prod.borelSpace [SecondCountableTopologyEither α β] :
   ⟨le_antisymm prod_le_borel_prod OpensMeasurableSpace.borel_le⟩
 #align prod.borel_space Prod.borelSpace
 
+proof_wanted ULift.instBorelSpace [BorelSpace α] : BorelSpace (ULift α)
```

Comment:
#8958 will probably help

Reply:
Yep, good point. Am happy to leave it like that for now, though. PFR only needs the `MeasurableSpace` instance, not the `BorelSpace` one.

Reply:
I'd lean towards splitting the lift stuff to a separate PR and doing it more exhaustively there

Reply:
Done